bancor_research/bancor_simulator/v3/spec/network.py

```diff
@@ -260,9 +256,11 @@ def begin_cooldown(
         state, tkn_name, user_name = validate_input(
             state, tkn_name, user_name, timestamp
         )
-        tkn_amt = userAmount(state, "bn" + tkn_name, user_name, tkn_amt)
+        tkn_amt = userAmount(
```

Comment:
I thought we/I changed all the naming from camelcase to lowercase for this function (at least in the simulator), maybe never got merged? anyway, for consistency would appreciate added/modified code to use the same lowercase naming conventions (e.g. `user_amount(...)`)

Reply:
Oh... Function `userAmount` is a very recent function which I have added; I'll sort this out.

Reply:
Second thought - I will rename this function (as well as function `toDecimal`) on a separate PR, in order to keep this one as clean as possible and allow us to carefully review the **functional** changes in it.